stream-chat-android-client/src/test/java/io/getstream/chat/android/client/testing/utils.kt

```diff
@@ -13,7 +13,7 @@ private val resClass = ResClass()
  * [path] "/model.member.json"
  */
 internal fun loadResource(path: String): String {
-    return convert(resClass.javaClass.getResourceAsStream(path))
+    return convert(requireNotNull(resClass.javaClass.getResourceAsStream(path)))
```

Comment:
AGP 4.1 enforces `@RecentlyNullable` annotations in a stricter way, hence these kinds of changes.